discord/flags.py

```diff
@@ -281,6 +281,13 @@ def join_notification_replies(self):
         .. versionadded:: 2.0
         """
         return 8
+    @flag_value
+    def app_commands_badge(self):
+        """:class:`bool`: Returns ``True`` if the application has registered a global application
+        command. This shows up as a badge in the the bot's profile.
+         .. versionadded:: 2.0
+        """
+        return 1 << 23
```

Comment:
Add empty lines, fix versionadded and remove typo :)
```suggestion

    @flag_value
    def app_commands_badge(self):
        """:class:`bool`: Returns ``True`` if the application has registered a global application
        command. This shows up as a badge in the bot's profile.

         .. versionadded:: 2.1
        """
        return 1 << 23
```

Reply:
updated